code/model/modelrender.h

```diff
@@ -74,6 +75,9 @@ class model_render_params
 	int Insignia_bitmap;
 
 	int *Replacement_textures;
+	bool manage_replacement_textures; // This is set when we are rendering a model without an associated ship object;
```

Comment:
Since this class now manages memory on its own it will likely cause either a memory leak or a double free if it is copied so we either need to delete the copy operators or properly implement them.